pytensor/tensor/nlinalg.py

```diff
@@ -582,26 +585,87 @@ def infer_shape(self, fgraph, node, shapes):
         else:
             return [s_shape]
 
+    def L_op(
+        self,
+        inputs: Sequence[Variable],
+        outputs: Sequence[Variable],
+        output_grads: Sequence[Variable],
+    ) -> list[Variable]:
+        (A,) = inputs
+        A = cast(ptb.TensorVariable, A)
```

Comment:
Why not define already `inputs: Sequence[TensorVariable]`?

Reply:
mypy complains, because the `Op` super class defines these as `Sequence[Variable]`.

Reply:
of course it does